### ISSUE TITLE
More of a question

### ISSUE THREAD

**Issue by dr3adx** [pixijs/pixi-particles#121]
Hello,

I'm using pixi particles to make ability that looks like shield around player in my game. I only render players that are near me, not all of them.
So my question is when new player enters my viewport and supposedly he had pixi particle (shield) running on him for 2 seconds before he entered my viewport and only now my client renders the particle emitter, is there a way to SKIP pixi particle 2 seconds in? As to show that specific duration has passed.
I was thinking of solutions to this, but even if I know that player had shield on him for 2 seconds i cant just reduce the duration of emitter by 2 secs because the animation will start from beginning (e.g lets take that megaman death example, i dont want to see circles going from very begining from inside to outside but rather show animation 2 seconds in when circles are all in outer layer)
To summarize, is it possible to skip-forward particles "X" amount of seconds in?

Thanks in advance.

**Comment by andrewstart** [comment]
call `yourEmitter.advance(2)` to run the emitter for 2 seconds immediately.

**Comment by dr3adx** [comment]
> call `yourEmitter.advance(2)` to run the emitter for 2 seconds immediately.

holy shiet you're da MVP bro!!!!!

**Comment by dr3adx** [comment]
@andrewstart 
Hello, I tried advance but it seems like there's no such function
`emitter.advance is not a function`

**Comment by andrewstart** [comment]
Yep, I should have checked the code instead of assuming that I remembered correctly - the method name is `update`, not `advance`.
